cvmfs/cvmfs_config

```diff
@@ -245,6 +312,12 @@ cvmfs_chksetup() {
     fi
   done
 
+  # Check that CVMFS_USER is set
+  if [ x"$CVMFS_USER" = x"" ]; then
```

Comment:
I've been running the script without having `/usr/local/etc/cvmfs` symlinked into `/etc/cvmfs` and added this check for convenience. Its not strictly necessary but I find it doesn't hurt either.